package.json

```diff
@@ -115,7 +115,7 @@
     "@wordpress/icons": "^2.9.0",
     "@wordpress/keycodes": "^2.16.0",
     "@wordpress/scripts": "^12.2.0",
-    "@wordpress/url": "^2.17.0",
+    "@wordpress/url": "^3.1.1",
```

Comment:
The version we had didn't contain `getQueryArgs` - seems like a reasonably recent addition.

Reply:
Looks like it was added later in 2.x although it isn't in the [changelog](https://github.com/WordPress/gutenberg/blob/trunk/packages/url/CHANGELOG.md). 3.x is mostly just removing support for IE 11. This should be fine though since we don't support IE 11 either 😄